triton-docker

```diff
@@ -14,18 +14,6 @@ function checkCurlInstalled () {
     command -v curl >/dev/null 2>&1 || { echo >&2 "curl is required to complete this installation. Please install before continuing."; exit 1; }
 }
 
-# Emit the current Triton profile and time
```

Comment:
Just moving this down so it's grouped with the other function related to environment checking/setting